code_schemes/s01e09.json

```diff
@@ -0,0 +1,376 @@
+      },
+      {
+        "CodeID": "code-0ac18866",
+        "CodeType": "Normal",
+        "DisplayText": "how to prevent",
+        "NumericValue": 4,
+        "StringValue": "how_to_prevent",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-e660f22d",
+        "CodeType": "Normal",
+        "DisplayText": "how to treat",
+        "NumericValue": 5,
+        "StringValue": "how_to_treat",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-5f42bc81",
+        "CodeType": "Normal",
+        "DisplayText": "government response",
+        "NumericValue": 6,
+        "StringValue": "government_response",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-8d5a049e",
+        "CodeType": "Normal",
+        "DisplayText": "somalia update",
+        "NumericValue": 7,
+        "StringValue": "somalia_update",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-3f99896b",
+        "CodeType": "Normal",
+        "DisplayText": "rumour/stigma/misinfo",
+        "NumericValue": 8,
+        "StringValue": "rumour_stigma_misinfo",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-dd59dae4",
+        "CodeType": "Normal",
+        "DisplayText": "collective hope",
+        "NumericValue": 9,
+        "StringValue": "collective_hope",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-46ac795e",
+        "CodeType": "Normal",
+        "DisplayText": "anxiety/panic",
+        "NumericValue": 10,
+        "StringValue": "anxiety_panic",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-c5dd8028",
+        "CodeType": "Normal",
+        "DisplayText": "religious hope/practice",
+        "NumericValue": 11,
+        "StringValue": "religious_hope_practice",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-2da70be2",
+        "CodeType": "Normal",
+        "DisplayText": "denial",
+        "NumericValue": 12,
+        "StringValue": "denial",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-89ae0d4f",
+        "CodeType": "Normal",
+        "DisplayText": "urgent need",
+        "NumericValue": 13,
+        "StringValue": "urgent_need",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-0634c3d7",
+        "CodeType": "Normal",
+        "DisplayText": "(call for) awareness creation",
+        "NumericValue": 14,
+        "StringValue": "call_for_awareness_creation",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-9ba88f73",
+        "CodeType": "Normal",
+        "DisplayText": "call for right practice",
+        "NumericValue": 15,
+        "StringValue": "call_for_right_practice",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-d55678d1",
+        "CodeType": "Normal",
+        "DisplayText": "rumour/stigma/misinfo - the virus discriminates",
```

Comment:
Did you mean to copy these L4 codes into the new schemes?

Reply:
Yes, this have become standards for Imaqal but not yet for Unicef.